Gems/EMotionFX/Code/Source/Editor/Plugins/Ragdoll/JointSwingLimitManipulators.cpp

```diff
@@ -0,0 +1,245 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzFramework/Physics/Configuration/JointConfiguration.h>
+#include <AzFramework/Viewport/ViewportColors.h>
+#include <AzFramework/Viewport/ViewportConstants.h>
+#include <AzToolsFramework/Manipulators/ManipulatorManager.h>
+#include <AzToolsFramework/Manipulators/ManipulatorView.h>
+#include <EMotionFX/CommandSystem/Source/JointLimitCommands.h>
+#include <EMotionStudio/EMStudioSDK/Source/EMStudioManager.h>
+#include <Editor/Plugins/Ragdoll/JointLimitRotationManipulators.h>
+#include <Editor/Plugins/Ragdoll/JointSwingLimitManipulators.h>
+#include <Editor/Plugins/Ragdoll/RagdollJointLimitWidget.h>
+
+namespace EMotionFX
+{
+    static constexpr float ManipulatorOffsetX = 0.2f; // manipulator position offset along the joint parent frame's X axis
+    static constexpr float ManipulatorScale = 400.0f; // scaling factor between linear position of manipulator and swing limit in degrees
+    static constexpr float ManipulatorInverseScale = 1.0f / ManipulatorScale;
```

Comment:
suggestion: Again AZ_CVARS might not be a bad idea (unless you're sure these are unlikely to change/be iterated on)